### DISCUSSION TITLE
Monokai GTK 3 Theme for changing the Geany UI on Windows 10/11

### DISCUSSION THREAD

**Discussion by zhaolinlau** [geany/geany#3283] [Show and tell]
Hey guys, for those who using Geany on Windows OS and want a dark theme ui like monokai theme, you can just clone my repo [Geany-WebDev-Snippets](https://github.com/zhaolinlau/Geany-WebDev-Snippets) and go to `Geany-WebDev-Snippets -> themes -> monokai`, then drag the whole `gtk-3.0` folder to your `C:\Users\Win10\AppData\Local`. 

After that, your Geany will be transformed from this
![image](https://user-images.githubusercontent.com/68846168/189481488-72c07300-a42f-475d-9d9a-bfd70292d45e.png)

to this
![image](https://user-images.githubusercontent.com/68846168/189481515-fbc8bb4e-20bd-4b87-b0cf-442a53959b7a.png)

And I will add more themes to my repo in the future.

**Comment by Andy2No** [comment]
This appears to be out of date.  I don't see a themes folder in the repo Geany-WebDev-Snippets, which has now been archived.

**Reply by zhaolinlau** [reply]
@Andy2No hi, i removed the theme folder because it no longer works in the geany v2. But if you're using geany<=1.38, you still can navigate to the older commit to copy the theme file.

**Reply by zhaolinlau** [reply]
@Andy2No btw, i already switched to vscode, no longer use geany.

**Comment by Andy2No** [comment]
@zhaolinlau Okay.  Thanks.
